### PR TITLE
Disable some tests

### PR DESCRIPTION
**Pull request by suhr**
Fixes https://github.com/mathics/Mathics/issues/946.

### CODE REVIEW

**Review by rocky — COMMENTED** [review]
It occurred to me later that it might be slicker to add a new indicator like `S>` to indicate that the test should be skipped _only_ in a sandboxed environment. 

`doc/doc.py`'s DocTest would be the place to change I think. Are you up for it?

**Comment by mmatera** [comment]
Yep: loot at mathics/doc/doc.py  line 1110

**Comment by suhr** [comment]
I added `S>` as a synonym to `X>`. I'm not going to implement it properly in this PR.

**Comment by suhr** [comment]
Feel free to implement it yourself though!

**Comment by mmatera** [comment]
I think it should be easy to replace all X> by S> in the code, and have just a unique notation...

**Comment by suhr** [comment]
Yeah, these tests use `S>` now.

**Review by rocky — APPROVED** [review]
LGTM - thanks for doing this!

File: mathics/doc/doc.py [hunk 1]
```diff
@@ -1124,7 +1125,7 @@ def __init__(self, index, testcase):
         # Private test cases are executed, but NOT shown as part of the docs
         self.private = testcase[0] == '#'
         # Ignored test cases are NOT executed, but shown as part of the docs
-        if testcase[0] == 'X':
+        if testcase[0] == 'X' or testcase[0] == 'S':
```
Comment (rocky) [inline mathics/doc/doc.py:1128]: I hit merge too soon. Shouldn't this be 

```
or (testcase[0] == 'S' and os.environ.get("SANDBOXED", False))
```

**Review by rocky — COMMENTED** [review]
Ooops noiticed that there's no way to have the test fire here. Se comment below.

**Pull request merged**